TACC/Core-CMS-Custom
# Admin Extensions React Conversion: Filters, Pagination, Modals

## Overview

This should (almost) complete the react conversion for the admin extensions listing page. Pagination, filters, and modals work. Minor issues that remain: clearing filters does not always reset them to "All", and closing out the edit modal does not refetch data. I was unable to find a solution to these and would appreciate suggestions. 

## Related

- [WP-662](https://tacc-main.atlassian.net/browse/WP-662)

## Changes
- Pagination implemented
- Filters (status and org) implemented
- View and Edit modals implemented

## Testing

1. Navigate to http://localhost:8000/administration/list-extensions/
2. Verify that Status and Org filters filter results as expected
3. Check "Clear Options" button works as expected (known issue: filter drop downs will sometimes not reset to "All")
4. Check that View action works as expected
5. Check that Edit action works as expected, including testing edits (known issue: listing will not update to reflect changes until manual refresh)
6. Check that Pagination works as expected

## UI

…
